etc/taskcluster/macos/README.md

```diff
@@ -42,20 +42,28 @@ This issue is tracked at
 
 * Log in through VNC, and run `xcode-select --install`
 
+* Still in VNC, install the jdk8 package from http://adoptopenjdk.net
```

Comment:
Would https://github.com/AdoptOpenJDK/homebrew-openjdk be appropriate, to automate this step in tasks rather than manual upfront setup?

r+ if you prefer to keep this as-is.

Reply:
I spent a long time trying to automate it yesterday, but the Brew formula involves sudo which seems impossible in our current salted setup.

Reply:
Ok. (We could run tasks as root or with sudo privileges, but it’s probably preferable not to.)

Reply:
Homebrew doesn't allow you to install things under an account that doesn't own the homebrew directory. It just got more and more complicated each time I tried to work around the latest issue.